pkg/supportbundle/supportbundle.go

```diff
@@ -173,701 +164,176 @@ func GetBundleCommand(appSlug string) []string {
-				Timeout:       "10s",
-			},
-		})
-	}
-	return dbCollectors
-}
-
-func makeKotsadmCollectors() []*troubleshootv1beta2.Collect {
-	names := []string{
-		"kotsadm-postgres",
-		"kotsadm",
-		"kotsadm-operator",
-		"kurl-proxy-kotsadm",
-		"kotsadm-dex",
-		"kotsadm-fs-minio",
-		"kotsadm-s3-ops",
-	}
-	kotsadmCollectors := []*troubleshootv1beta2.Collect{}
-	for _, name := range names {
-		kotsadmCollectors = append(kotsadmCollectors, &troubleshootv1beta2.Collect{
-			Logs: &troubleshootv1beta2.Logs{
-				CollectorMeta: troubleshootv1beta2.CollectorMeta{
-					CollectorName: name,
-				},
-				Name:      "kots/admin-console",
-				Selector:  []string{fmt.Sprintf("app=%s", name)},
-				Namespace: os.Getenv("POD_NAMESPACE"),
-			},
-		})
-	}
-	return kotsadmCollectors
-}
-
-func makeGoRoutineCollectors() []*troubleshootv1beta2.Collect {
-	names := []string{
-		"kotsadm",
-		"kotsadm-operator",
-	}
-	goroutineCollectors := []*troubleshootv1beta2.Collect{}
-	for _, name := range names {
-		goroutineCollectors = append(goroutineCollectors, &troubleshootv1beta2.Collect{
-			Exec: &troubleshootv1beta2.Exec{
-				CollectorMeta: troubleshootv1beta2.CollectorMeta{
-					CollectorName: fmt.Sprintf("%s-goroutines", name),
-				},
-				Name:          "kots/admin-console",
-				Selector:      []string{fmt.Sprintf("app=%s", name)},
-				Namespace:     os.Getenv("POD_NAMESPACE"),
-				ContainerName: name,
-				Command:       []string{"curl"},
-				Args:          []string{"http://localhost:3030/goroutines"},
-				Timeout:       "10s",
-			},
-		})
-	}
-	return goroutineCollectors
-}
-
-func makeRookCollectors() []*troubleshootv1beta2.Collect {
-	names := []string{
-		"rook-ceph-agent",
-		"rook-ceph-mgr",
-		"rook-ceph-mon",
-		"rook-ceph-operator",
-		"rook-ceph-osd",
-		"rook-ceph-osd-prepare",
-		"rook-ceph-rgw",
-		"rook-discover",
-	}
-	rookCollectors := []*troubleshootv1beta2.Collect{}
-	for _, name := range names {
-		rookCollectors = append(rookCollectors, &troubleshootv1beta2.Collect{
-			Logs: &troubleshootv1beta2.Logs{
-				CollectorMeta: troubleshootv1beta2.CollectorMeta{
-					CollectorName: name,
-				},
-				Name:      "kots/rook",
-				Selector:  []string{fmt.Sprintf("app=%s", name)},
-				Namespace: "rook-ceph",
-			},
-		})
-	}
-
-	rookCollectors = append(rookCollectors, &troubleshootv1beta2.Collect{
-		Ceph: &troubleshootv1beta2.Ceph{},
-	})
-
-	return rookCollectors
-}
-
-func makeKurlCollectors() []*troubleshootv1beta2.Collect {
-	names := []string{
-		"registry",
-	}
-	collectors := []*troubleshootv1beta2.Collect{}
-	for _, name := range names {
-		collectors = append(collectors, &troubleshootv1beta2.Collect{
-			Logs: &troubleshootv1beta2.Logs{
-				CollectorMeta: troubleshootv1beta2.CollectorMeta{
-					CollectorName: name,
-				},
-				Name:      "kots/kurl",
-				Selector:  []string{fmt.Sprintf("app=%s", name)},
-				Namespace: "kurl",
-			},
-		})
+		err = errors.Wrap(err, "failed to create temp dir")
+		logger.Error(err)
```

Comment:
Looks like errors in this function might be double-logged

Reply:
Leaving per our slack convo about this being existing behavior.